src/scripts/main.js

```diff
@@ -1,7 +1,52 @@
 'use strict';
 
-const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+const createNotification = (posTop, posRight, title, description, type) => {
+  const notification = document.createElement('div');
+
+  notification.className = `notification ${type}`;
+  notification.style.position = 'fixed';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.className = 'title';
+  titleElement.textContent = title;
+  notification.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+  notification.appendChild(descriptionElement);
+
+  return notification;
```

Comment:
It's a good practice to separate the UI logic from the business logic. Here, you are creating a notification which is more related to UI. It might be a good idea to move this function into a separate module, say 'ui.js', where you would keep all UI-related functions.